src/storage/txn/commands/rollback.rs

```diff
@@ -60,3 +82,94 @@ impl<S: Snapshot, L: LockManager> WriteCommand<S, L> for Rollback {
         })
     }
 }
+
+// #[cfg(test)]
```

Comment:
It's quite strange that if I use `#[cfg(test)]`, the compiler will complain about `failed to resolve: could not find tests in rollback`.
And I cannot find out why or how can I solve this problem